### PR TITLE
Add Walkthrough 1A steps

### PR DESCRIPTION
**Pull request by JameelB**
## Description
Add steps for walkthrough 1A which uses Enmasse and Fuse.

### CODE REVIEW

**Comment by JameelB** [comment]
@finp Hope this gives some insight into walkthrough 1A.

File: walkthrough-1A/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Prerequisites
+
+- The installation has already been performed on an OpenShift cluster as per the installation document https://github.com/integr8ly/installation/blob/master/README.md.
+
+# Setting up the Enmasse Address
```
Comment (aidenkeating) [inline walkthrough-1A/README.md:5]: @JameelB Perhaps we should have a set before this about selecting OpenShift on the right-hand side as the login option when brought to the auth page for EnMasse. This seems to confuse a lot of people.

File: walkthrough-1A/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Open the `Enmasse Console` and go to the Addresses tab. Create a new address, 
+name it `work-queue/requests`, select `queue` as the type and click `Next`.
```
Comment (aidenkeating) [inline walkthrough-1A/README.md]: Just a note that creating a new address means is clicking the `Create` button at the top of the page (centered, if that matters).

File: walkthrough-1A/README.md (hunk 1 above)
Comment (JameelB) [reply walkthrough-1A/README.md:5]: I can change the first sentence to: `Open the Enmasse Console, click Openshift and login using your Openshift credentials` Wdyt?

**Review by aidenkeating — APPROVED** [review]
Few small comments, but verified on my cluster.

**Pull request merged**